site/docs/plugins/keyboard.md

```diff
@@ -190,44 +170,27 @@ const keyboard = new Keyboard()
   .resized();
 ```
 
-##### Result
-
-![Example 1](/images/keyboard-example-1.webp)
+You can also send more powerful buttons that request the user's phone number or location or do other cool things.
+Be sure to check out [all methods](https://deno.land/x/grammy/mod.ts?s=Keyboard#Methods) on the `Keyboard` class.
 
-#### Example 2
+If you already have an array of strings that you would like to turn into a keyboard, you can use a second, alternative style for building keyboard instances.
```

Comment:
We have exactly the same text for inline keyboard and for custom keyboards. There is any manner to reuse it like putting in some section or ...?

Reply:
The assumption here is that people only read the top part of the page, then decide which keyboard type they want, and only read one of the two parts: either inline keyboards or custom keyboards. The same text is required in two locations because the same concepts applies for both keyboard types. (In the actual code, the logic is shared.)